pkg/dev/dev.go

```diff
@@ -327,11 +301,6 @@ func (m *Manager) InitializeRelayCore(ctx context.Context, initOpts InitializeOp
 		}
 	}
 
-	if err := mm.ProcessManifests(ctx, "/01-init",
```

Comment:
Should these be renumbered at some point? Or is the numbering even necessary since they seem like they're explicitly being applied in order?

Reply:
I'm going to remove the numbering (I've already started with newer additions, and plan to reorganize the folders).
Trying to avoid making too many changes to both files and folders at once, to avoid unnecessary complexity in the code differences.

Reply:
I'll probably just apply that as a trivial change after all the latest changes are in.